lib/l10n/app_en.arb

```
@@ -1757,5 +1769,89 @@
+  "description": "Text used when inviting friends to the app.",
+  "placeholders": {
+    "url": {
+      "type": "String"
     }
+  }
+},
+"@thisMessageWasDeleted": {
+  "description": "Status text shown when a previously sent message has been deleted."
+},
+
+"failedToDeleteMessage": "Failed to delete message",
```

RuboCop (1.84.2)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)